docker/prestart_core.sh

```diff
@@ -54,8 +54,8 @@ elif [[ -n "$TARANIS_NG_ADMIN_USERNAME" && -n "$TARANIS_NG_ADMIN_PASSWORD" ]]; t
         CONFIG_PUBLISHER_PRESET_ACCESS, CONFIG_PUBLISHER_PRESET_CREATE, \
         CONFIG_PUBLISHER_PRESET_UPDATE, CONFIG_PUBLISHER_PRESET_DELETE" || true
 
-    ROLE_ID=$(/app/manage.py role --list --filter "Admin role" | grep -i id | awk '{print $2}')
+    ROLE_ID="$(/app/manage.py role --list --filter "Admin role" | grep -i id | awk '{print $2}')"
```

Comment:
this is incorrect, since the quotes are also used internally. Fortunately this code is just a placeholder that has never been used so far. Please rewrite and test.

Reply:
see my comment below